src/GovUk.Education.ExploreEducationStatistics.Common/Utils/EnumUtil.cs

```diff
@@ -46,4 +89,22 @@ private static class EnumCache<TEnum> where TEnum : Enum
     {
         return GetEnums<TEnum>().ToArray();
     }
+
+    private static IEnumerable<string> GetEnumValuesUncached<TEnum>() where TEnum : Enum
+        => GetEnums<TEnum>().Select(e => e.GetEnumValue());
+
+    public static IReadOnlyList<string> GetEnumValues<TEnum>() where TEnum : Enum
+        => EnumCache<TEnum>.ValuesList.Value;
+
+    public static IReadOnlySet<string> GetEnumValuesSet<TEnum>() where TEnum : Enum
+        => EnumCache<TEnum>.ValuesSet.Value;
+
+    private static IEnumerable<string> GetEnumLabelsUncached<TEnum>() where TEnum : Enum
+        => GetEnums<TEnum>().Select(e => e.GetEnumLabel());
+
+    public static IReadOnlyList<string> GetEnumLabels<TEnum>() where TEnum : Enum
+        => EnumCache<TEnum>.LabelsList.Value;
+
+    public static IReadOnlySet<string> GetEnumLabelsSet<TEnum>() where TEnum : Enum
+        => EnumCache<TEnum>.LabelsSet.Value;
```

Comment:
Might be best to order these so that `private` methods are together, and `public` methods are together. Just makes it slightly more readable IMO